mapviz/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mapviz)
 
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS} -D__STDC_FORMAT_MACROS")
```

Comment:
Consider making these flag changes local to the mapviz target instead of global to all packages in the workspace:

set_target_properties(mapviz PROPERTIES COMPILE_FLAGS "-std=c++11 -D__STDC_FORMAT_MACROS")